css/styles.css

```diff
@@ -58,15 +60,18 @@
     margin: 0;
     font-size: 14px;
     font-weight: normal;
+    line-height: 20px;
     text-transform: uppercase;
 }
 .ab-column h3.icon:before {
-    font: normal 28px/28px icons;
+    font: normal 30px/30px icons;
```

Comment:
Due to WordPress' usage of a native font-stack and/or different rendering by different browsers the outcome of this might vary. I agree the 30px/30px do not produce in a sub-pixel perfect result, but at least in my tests 28px/28px result in a slightly worse alignment.